faster_whisper/utils.py

```diff
@@ -21,6 +21,7 @@
     "large-v1": "guillaumekln/faster-whisper-large-v1",
     "large-v2": "guillaumekln/faster-whisper-large-v2",
     "large": "guillaumekln/faster-whisper-large-v2",
+    "large-v3": "gradjitta/ct2-whisper-large-v3",
```

Comment:
This repo is not official, was not generated from the official HF openai/whisper-large-v3 repo and the tokenizer.json is likely wrong, there is no yue in the languages.

Reply:
which would you suggest using? @bungerr has one at `bababababooey/faster-whisper-large-v3` but didn't copy the `preprocessor_config.json` over, which is useful to get the feature extractor setup correctly with.

Reply:
oop i just added it in, had to copy the alignment heads over from there & forgot

also accidentally wiped my local openai/whisper-large-v3 repo, pulling again & will reconvert when the new wheels finish building https://github.com/OpenNMT/CTranslate2/actions/runs/6801730648

Reply:
note: tokenizer.json is not provided in the official repo so if you want to run the ct2 translation script, you'll have to either use hf's openai_to_hf converter script (which may have been updated since i last checked(?)), or just do what i did and load+save from AutoTokenizer

```
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("openai/whisper-large-v3")
tokenizer.save_pretrained("./tokenizerjson")
```